.github/workflows/helm-vib.yaml

```diff
@@ -1,7 +1,7 @@
 name: Verify Helm Chart
 on:
   workflow_dispatch:
-  pull_request_target:
+  push:
```

Comment:
I am a little bit confuse, why we are changing from pull_request_target to push? I mean, we are going to launch this VIB action every time that we are pushing, no? Am I wrong?

Reply:
No, it only triggers when you **push** to `main`, and that ONLY happens on merges, as `main` is protected.

Reply:
I think this is a leftover:

```
bitnami-labs:main
```

Just:

```
branches:
  - main
```

Reply:
fixed